spec/spec.md

```diff
@@ -113,9 +139,13 @@ Presentation Definitions are objects generate to articulate what proofs an entit
               "pattern": "^DE|^US|^JP"
             }
           }
+          // TODO: how do you express zero-knowledge predicate requirements?
```

Comment:
I think we need to see examples of the JSON that would be transmitted. I suggest starting by simply creating a credential of this format, and placing it next to the JWT and LD Proofs... if we can't provide a real world example of how this works with Indy Credentials, I think this spec is dead on arrival.

Reply:
raised issue #46 to enable discussion